admin/build.sbt

```diff
@@ -0,0 +1,46 @@
+lazy val piezoAdmin = (project in file(".")).enablePlugins(PlayScala)
+
+bashScriptExtraDefines ++= Seq(
+  "mkdir -m 664 /var/run/piezo-admin/",
+  s"chown ${(daemonUser in Linux).value} /var/run/piezo/admin",
+  s"chown ${(daemonGroup in Linux).value} /var/run/piezo/admin",
+  """JAVA_OPTS="-Dpidfile.path=/var/run/piezo-admin/piezo-admin.pid $JAVA_OPTS"""",
```

Comment:
That will not work. The quartz library comes with its own quartz.properties
file in that location in its jar.

On Mon, Aug 17, 2015 at 10:38 AM, Paul Draper <notifications@github.com>
wrote:

> In admin/build.sbt
> <https://github.com/lucidsoftware/piezo/pull/32#discussion_r37207553>:
>
> > @@ -0,0 +1,46 @@
> > +lazy val piezoAdmin = (project in file(".")).enablePlugins(PlayScala)
> > +
> > +bashScriptExtraDefines ++= Seq(
> > +  "mkdir -m 664 /var/run/piezo-admin/",
> > +  s"chown ${(daemonUser in Linux).value} /var/run/piezo/admin",
> > +  s"chown ${(daemonGroup in Linux).value} /var/run/piezo/admin",
> > +  """JAVA_OPTS="-Dpidfile.path=/var/run/piezo-admin/piezo-admin.pid $JAVA_OPTS"""",
>
> By default, it will use org/quartz/quartz.properties.
>
> I changed the file to that location (from its original quartz.properties)
> rather than set a default system property. I think that's better.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lucidsoftware/piezo/pull/32/files#r37207553>.
>